lua/typescript-tools/protocol/text_document/jsx_close_tag.lua

```diff
@@ -0,0 +1,21 @@
+local c = require "typescript-tools.protocol.constants"
+local utils = require "typescript-tools.protocol.utils"
+
+local M = {}
+
+---@type TsserverProtocolHandler
+function M.handler(request, response, params)
+  request(utils.tsserver_location_request(c.CommandTypes.JsxClosingTag, params))
+
+  local body = coroutine.yield()
+
+  -- if execute failed will back a response include success field.
+  if not body.success then
```

Comment:
I missed out `.success` is nil when req is successful so this condition is always true. My bad.
```suggestion
  if body.success == false then
```